### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by svuckovicTT**
- Update CODEOWNERS
- Added folks from "mlir core belgrade" team (@sdjordjevicTT @mtopalovicTT @rpavlovicTT @svuckovicTT) to relevant sections
- Ideally, we would have teams, such as "mlir-core-belgrade" team, then we could substitute above aliases with a single `@tenstorrent/mlir-core-belgrade` item and make it easier to read and understand _why_ someone is a codeowner
  - @warthog9 can we sync on this? I'd like to understand the overhead of managing this

### CODE REVIEW

**Review by nvukobratTT — COMMENTED** [review]
@sdjordjevicTT @svuckovicTT lets think about how we can split future TTIR related to decompositions. It'll be good for all frontend leads to one of the owners of breadth TTIR and decomposition logic as well :))

**Comment by svuckovicTT** [comment]
> @sdjordjevicTT @svuckovicTT lets think about how we can split future TTIR related to decompositions. It'll be good for all frontend leads to one of the owners of breadth TTIR and decomposition logic as well :))

Sure, though let's keep it outside the scope of this PR. I'd like to get this in now, so that the whole team is aware of the changes happening.

File: .github/CODEOWNERS
Comment (tt-mpantic) [inline .github/CODEOWNERS:1]: We were mentioning removing root approvers on F2F. Should we do that now ?
@nsmithtt do you agree ?

Comment (tt-mpantic) [reply .github/CODEOWNERS:1]: I would expect that many folks would like to be subscribed to all PRs and getting the same value that you are talking about.
My understanding is that main intention of slack PRs channel #tt-mlir-code-changes was to cover that exact scenario.
Should we fix something with the PRs channel if current implementation is not enough (I know that rest of the team, including me, are using that Slack channel as a source of all PRs)?

Comment (svuckovicTT) [reply .github/CODEOWNERS:1]: Would something like this work? In the next UI component, one can check PRs, I _assume_ all PRs would get mailed.

image

Comment (nsmithtt) [reply .github/CODEOWNERS:1]: @svuckovicTT, yeah I think that works, much prefer email over the slack channel because the slack channel doesn't have any way to mark which PRs I've reviewed.

Comment (svuckovicTT) [reply .github/CODEOWNERS]: Should be good now.

**Review by nsmithtt — APPROVED** [review]

**Review by vmilosevic — APPROVED** [review]
Looks good

File: .github/CODEOWNERS
Comment (nsmithtt) [reply .github/CODEOWNERS]: Thanks! Oh can you also add me to all cmake files & python:
```
*.txt
*.cmake
/python/
```

**Comment by sdjordjevicTT** [comment]
Can we merge this? The Metal uplift is blocked on Rade's(currently out of office) sign-off as he was the only reviewer for TTKernel stuff:
https://github.com/tenstorrent/tt-mlir/pull/870

**Pull request merged**
